src/gfx/main.cpp

```diff
@@ -364,6 +366,43 @@ static char *parseArgv(int argc, char *argv[]) {
 			}
 			options.attrmap = musl_optarg;
 			break;
+		case 'B':
+			if (!strcmp(musl_optarg, "transparent") || !strcmp(musl_optarg, "TRANSPARENT")) {
```

Comment:
Alternatively, use `strcasecmp`? That'd be less code and also support `Transparent`. (I know it permits SpOnGeBoBcAsE, but if someone wants to put them through that... let them be?)

Reply:
`strcasecmp` is in POSIX, not the C standard. Is [`_stricmp`](https://stackoverflow.com/a/26359433/2738262) good enough on Windows?

Reply:
Yes, [we have an alias definition](https://github.com/gbdev/rgbds/blob/56f72222302b160f2e44b34750adb7e63294f99c/include/platform.hpp#L8-L15).